benchmarks/qr-code/index.ts

```diff
@@ -0,0 +1,151 @@
+import QrCode from "qrcode";
+import { wrapDocument } from "../../src/index";
+import { encode as cborEncode } from "cbor";
+import { gzipSync } from "zlib";
+
+type IOverload = {
+  (field: string, defaultValue: string): string;
+  (field: string, defaultValue: number): number;
+};
+/**
+ * return value from argument. It looks for an existing key, prepended with -- and return the following value associated
+ * otherwise return the default value
+ */
+const getArgument: IOverload = (field: string, defaultValue: any): any => {
+  const index = process.argv.findIndex(value => value === `--${field}`);
+  if (index !== -1) {
+    const value = process.argv[index + 1];
+    if (typeof defaultValue === "number") return parseFloat(value);
+    return value;
+  }
+  return defaultValue;
+};
+
+/**
+ * Utility function to generate string for a predefined size
+ */
+const latinCharacters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789 ";
+const chineseCharacters =
+  "的一不是了人我在有他這中大來上國個到說們為子和你地出道也時年得就那要下以生會自著去之過家學對可她裡后 一二三四五六七八九十";
+function makeString(length: number, characters: string) {
+  const result = [];
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result.push(characters.charAt(Math.floor(Math.random() * charactersLength)));
+  }
+  return result.join("");
+}
+
+/**
+ * Minimum set of data for oa document. $template is not mandatory, but it's a key feature and widely used
+ */
+const data: any = {
+  id: "SERIAL_NUMBER_123",
+  $template: {
+    name: "CUSTOM_TEMPLATE",
+    type: "EMBEDDED_RENDERER",
+    url: "https://localhost:3000/renderer"
+  },
+  issuers: [
+    {
+      name: "DEMO STORE",
+      documentStore: "0x9178F546D3FF57D7A6352bD61B80cCCD46199C2d",
+      identityProof: {
+        type: "DNS-TXT",
+        location: "tradetrust.io"
+      }
+    }
+  ]
+};
+
+/**
+ * wrap a document and automatically the fields with random data. data can contain any characters provided as an argument.
+ * Two pre-defined sets of characters exist, one with latin characters, one with chinese characters.
+ * the fields name is always using the latin characters set.
+ * every field name is 8 characters length, and every value is 20 characters length. It could be made configurable.
```

Comment:
for the field name, thats not configurable right?
the wording makes it seem like its configurable

Reply:
It's not indeed, which part make you think it is?